pkg/utils/release/releases.go

```diff
@@ -122,5 +122,5 @@ func (r *ReleaseController) GetPipelineRunInNamespace(namespace, releaseName, re
 		return &pipelineRuns.Items[0], nil
 	}
 
-	return nil, fmt.Errorf("couldn't find PipelineRun in managed namespace '%s' for a release '%s' in '%s' namespace", namespace, releaseName, releaseNamespace)
+	return nil, fmt.Errorf("couldn't find PipelineRun in managed namespace '%s' for a release '%s' in '%s' namespace because of err:'%s'", namespace, releaseName, releaseNamespace, err)
```

Comment:
When returning an error, it's a good idea to use `%w` to wrap the original error instead of `%s` e.g. 
```go
fmt.Errorf("... because of error: %w", ..., err)
```

Reply:
Thanks @lcarva and updated.